public/app/features/alerting/unified/components/receivers/form/GrafanaReceiverForm.tsx

```diff
@@ -82,16 +88,23 @@ export const GrafanaReceiverForm = ({ contactPoint, readOnly = false, editMode }
 
   const onSubmit = async (values: ReceiverFormValues<GrafanaChannelValues>) => {
     const newReceiver = formValuesToGrafanaReceiver(values, id2original, defaultChannelValues, grafanaNotifiers);
+
     try {
       if (editMode) {
-        await updateContactPoint({
-          contactPoint: newReceiver,
-          id: contactPoint!.id,
-          resourceVersion: contactPoint?.metadata?.resourceVersion,
-          originalName: contactPoint?.name,
-        });
+        if (useK8sAPI && contactPoint && contactPoint.id) {
```

Comment:
is necessary here to check if `useK8sAPI`?

Reply:
maybe it would be more readable creating a variable with this expression so we can understand what exactly we are checking?